<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainchat</class>
 <widget class="QMainWindow" name="Mainchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MamadGram - MainChat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/App_Data/Icons/Icon.png</normaloff>:/App_Data/Icons/Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="MessageBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>1321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/App_Data/Images/linear-messagebox.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Logout">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Logout
{
border-image: url(:/App_Data/Images/close_red.png);
}
#Logout:hover
{
border-image: url(:/App_Data/Images/close_red-clicked.png);
}
#Logout:pressed
{
border-image: url(:/App_Data/Images/close_red.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>221</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QListWidget" name="PV_ListWidget"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="Group_ListWidget"/>
       </item>
       <item>
        <widget class="QListWidget" name="Channel_ListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="PV_Options">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#PV_Options
{
border-image: url(:/App_Data/Images/user-group-chat-02-512.png);
}
#PV_Options:hover
{
border-image: url(:/App_Data/Images/user-group-chat-02-512-clicked.png);
}
#PV_Options:pressed
{
border-image: url(:/App_Data/Images/user-group-chat-02-512.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Channel_Join">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Channel_Join
{
border-image: url(:/App_Data/Images/satelite-radar-icon-vector.png);
}
#Channel_Join:hover
{
border-image: url(:/App_Data/Images/satelite-radar-icon-vector5143252532325.png);
}
#Channel_Join:pressed
{
border-image: url(:/App_Data/Images/satelite-radar-icon-vector.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GP_Join">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#GP_Join
{
border-image: url(:/App_Data/Images/joingroup.png);
}
#GP_Join:hover
{
border-image: url(:/App_Data/Images/join clicked.png);
}
#GP_Join:pressed
{
border-image: url(:/App_Data/Images/joingroup.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatbar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>630</y>
      <width>911</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type The Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendmessage">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>620</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#sendmessage
{
border-image: url(:/App_Data/Images/send.png);
}
#sendmessage:hover
{
border-image: url(:/App_Data/Images/send.clicked.png);
}
#sendmessage:pressed
{
border-image: url(:/App_Data/Images/send.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="join_id">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name For Joining Group or Channel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="make_id">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name for Creating\Joining Group Or Channel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PV_usernamelabel_startchat">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>PV User for Sending Message</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>1011</width>
      <height>551</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GP_Make">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#GP_Make
{
border-image: url(:/App_Data/Images/add_group-512.png);
}
#GP_Make:hover
{
border-image: url(:/App_Data/Images/add_group-512-clicked.png);
}
#GP_Make:pressed
{
border-image: url(:/App_Data/Images/add_group-512.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Channel_Create">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Channel_Create
{
border-image: url(:/App_Data/Images/satelite-radar-icon-vector123213231.png);
}
#Channel_Create:hover
{
border-image: url(:/App_Data/Images/satelite-radar-icon-vector-clicked-join.png);
}
#Channel_Create:pressed
{
border-image: url(:/App_Data/Images/satelite-radar-icon-vector123213231.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>610</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="Group_Channel_Title">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter The Group/Channel Title</string>
    </property>
   </widget>
   <zorder>MessageBox</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>PV_Options</zorder>
   <zorder>Channel_Join</zorder>
   <zorder>GP_Join</zorder>
   <zorder>chatbar</zorder>
   <zorder>sendmessage</zorder>
   <zorder>Logout</zorder>
   <zorder>join_id</zorder>
   <zorder>make_id</zorder>
   <zorder>PV_usernamelabel_startchat</zorder>
   <zorder>textBrowser</zorder>
   <zorder>GP_Make</zorder>
   <zorder>Channel_Create</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>Group_Channel_Title</zorder>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
